ntoskrnl/include/internal/mm.h

```diff
@@ -866,6 +866,36 @@ NTAPI
 MmDeleteKernelStack(PVOID Stack,
                     BOOLEAN GuiStack);
 
+/* balance.c / pagefile.c******************************************************/
+
+inline VOID UpdateTotalCommittedPages(LONG Increase)
```

Comment:
Can you call that `Delta` or similar please ? Negative increase  is not an increase anymore 🙂

Reply:
Done

Reply:
Thanks, I was struggling to come up with a better name haha